utility/unit_of_work/block.go

```diff
@@ -12,48 +12,67 @@ import (
 	typesUtil "github.com/pokt-network/pocket/utility/types"
 )
 
-func (u *baseUtilityUnitOfWork) beginBlock(previousBlockByzantineValidators [][]byte) typesUtil.Error {
-	if err := u.handleByzantineValidators(previousBlockByzantineValidators); err != nil {
+func (uow *baseUtilityUnitOfWork) beginBlock() typesUtil.Error {
+	log := uow.logger.With().Fields(map[string]interface{}{
+		"source": "beginBlock",
+	}).Logger()
+
+	log.Debug().Bool("TODO", true).Msg("determining prevBlockByzantineValidators")
```

Comment:
Should we put this log in the helper function?

Reply:
💅 since it's empty I would say no. That would push me into adding log statements in all the utility functions as well and I wanted to keep the scope tight all considered :)